addons/armory/initSettings.hpp

```diff
@@ -0,0 +1,21 @@
+[
+    QGVAR(cba_settings_actions),
+    "LIST",
+    [LLSTRING(cba_settings_actions), LLSTRING(cba_settings_actions_tooltip)],
+    [LELSTRING(main,category), LLSTRING(subCategory)],
+    [["ace_interaction","scroll_menu"],[LLSTRING(cba_settings_actions_ace_interaction), LLSTRING(cba_settings_actions_scroll_menu)],0],
+    1,
+    {}
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(cba_settings_playerDBConnection),
```

Comment:
```suggestion
    QGVAR(allowPlayerDBConnection),
```

Reply:
Var is renamed to allowPlayerDBConnection